aws-datastore/src/test/java/com/amplifyframework/datastore/AWSDataStorePluginTest.java

```diff
@@ -61,14 +69,14 @@
      * the SyncProcessor is running. Otherwise, either the SyncProcessor is *not*
      * running, or it is running but not functioning as we expect it to.
      */
+    @SuppressWarnings("unchecked")
```

Comment:
```suggestion
```
I'd expect it to be on the class, though -- or on one of the `@Test`. Here one the `@Before`, it doesn't seem like it suppressing anything inside of this particular method.